tools/copydlldeps.md

```diff
@@ -45,6 +45,24 @@ if [ ! $( echo $compiler | grep -q "shared" ) ]; then
 fi
 ```
 
+Test & example
+==============
+
+Let us compile password manager [KeePassX](https://www.keepassx.org) in
+shared mode (i686-w64-mingw32.shared) and gather all its DLL dependencies.
+
+```bash
+$ make keepassx MXE_PLUGIN_DIRS=plugins/apps \
+    MXE_TARGETS='i686-w64-mingw32.shared'
+$ ./tools/copydlldeps.sh --copy --destdir keepassx-bundle \
+    --infile usr/i686-w64-mingw32.shared/bin/KeePassX.exe \
+    --recursivesrcdir usr/i686-w64-mingw32.shared/
+$ cp -r usr/i686-w64-mingw32.shared/bin/{KeePassX.exe,share} keepassx-bundle
```

Comment:
@starius - I have had an idea of how to easily add a neat script to test and teach people of how to use copydlldeps.sh
https://github.com/dl5rcw/mxe/blob/master/tools/copydlldepsexample.sh

Basically, it makes use of Qt5 itself. Qt has examples already delivered to the project. I found them stored in pkg directory of my working mxe environment.

It grabs the qtbase, copies to a tmp file that it created before, runs everything needed to build the package and delivers all dependencies to a OUTDIR directory. All you may want to modify is MXEDIR and OUTDIR in the beginning.
Have a look in the script and you may decided to either add it to the copydlldeps.md file, or put it to the tools directory. I think it can teach people how to use mxe, qt5 and deliver a package with all dependencies. I copied cube from my OUTDIR of the server to my desktop and it started right away. 

You may decide if you want to edit the lines to copydlldeps.md in a Test and Example section, or if you want to provide the bash script pure in tools.